15_1_baloon.js

```diff
@@ -0,0 +1,55 @@
+const baloon = document.querySelector('.baloon');
+const measure = document.querySelector('.measure');
+const btnStart = document.querySelector('.start');
+
+let size = 50;
+let level = -200;
+
+const playGame = () => {
+    clearScore();
+    window.addEventListener('keydown', pump);
+};
+
+const pump = event => {
+    if (event.keyCode === 38) {
```

Comment:
what is `38`? Add it to the `const` and name it appropriately like `upArrowKeyCode` or sth